TreeMaker/interface/TreeMaker.h

```diff
@@ -62,9 +62,9 @@ class TreeMaker : public edm::one::EDAnalyzer<edm::one::SharedResources> {
 		vector<TreeTypes> VarTypes;
 		map<string,unsigned> nameCache;
 		// general event information
-		UInt_t runNum;
-		UInt_t lumiBlockNum;
-		ULong64_t evtNum;
+		UInt_t runNum{};
```

Comment:
Why add brackets here? What does that do in a technical sense?

Reply:
see:
* https://clang.llvm.org/extra/clang-tidy/checks/cppcoreguidelines-pro-type-member-init.html
* http://en.cppreference.com/w/cpp/language/zero_initialization